rskj-core/src/main/java/co/rsk/util/TraceUtils.java

```diff
@@ -0,0 +1,14 @@
+package co.rsk.util;
+
+import org.apache.commons.lang3.RandomStringUtils;
+
+public class TraceUtils {
+    public static final String MSG_ID = "msg.id";
```

Comment:
```suggestion
    public static final String MSG_ID = "peerMsgId";
```

we already use the camel case in [here](https://github.com/rsksmart/rskj/blob/0fbd4bbbfa464d5031f09a622ce34778ac8327c3/rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java#L148), so I guess these ones should be consistent as well